Gemfile-legacy

```diff
@@ -0,0 +1 @@
+Gemfile
```

Comment:
Instead of symlinking, let''s get rid of the conditional logic in the main gemfile and just remove vernier from other legacy one. It ismore scalable as we add more gemfiles to the CI matrix. We should really be testing different Rails versions.

Reply:
ok